dask/array/core.py

```diff
@@ -5409,7 +5409,7 @@ def _vindex(x, *indexes):
 
     nonfancy_indexes = []
     reduced_indexes = []
-    for i, ind in enumerate(indexes):
+    for _, ind in enumerate(indexes):
```

Comment:
Since we're not using `i`, let's just drop the `enumerate(...)` call altogether 

```suggestion
    for ind in indexes:
```

Reply:
Nice suggestion! I got carried away focusing too narrowly. Added in.